### PR TITLE
Add `responsible_role` and parameters to `component` schema

### PR DESCRIPTION
**Pull request by jcscottiii**
Gitbook does not print out the parameters because it does not make sense to print them out.

Docx Template:
image

Filled in Template:
image

### CODE REVIEW

**Comment by jcscottiii** [comment]
Relies on #143

File: docx/docx.go [hunk 1]
```diff
@@ -31,13 +31,59 @@ func (config *Config) BuildDocx() error {
...
+				// Look if there's parameter with the same key as the passed in parameterKey.
+				found := false
+				var parameterText string
+				for _, parameter := range justification.SatisfiesData.Parameters {
+					// If section header exists, let's print it's corresponding text and not the header itself.
+					if parameter.Key == parameterKey {
+						parameterText = fmt.Sprintf("%s%s\n", parameterText, parameter.Text)
+						found = true
+					}
+				}
```
Comment (afeld) [inline docx/docx.go]: Mind making a `GetParameter` method?

File: docx/docx.go [hunk 2]
```diff
@@ -31,13 +31,59 @@ func (config *Config) BuildDocx() error {
...
+				// If we actually found a section, let's prepend the component name to the parameter.
+				if found {
+					text = fmt.Sprintf("%s%s:\n%s", text, component.Name, parameterText)
+				}
```
Comment (afeld) [inline docx/docx.go]: Ditto the question in #143 about what the behavior for `found == false` should be.

File: models/components_test.go [hunk 3]
```diff
@@ -27,31 +64,7 @@ var componentTests = []componentTest{
...
-						NarrativeSection{Key: "b", Text: "Justification in narrative form B for 1.1.1"},
-					},
-				},
-				{
-					Narrative: []NarrativeSection{
-						NarrativeSection{Text: "Justification in narrative form for 2.1"},
-					},
-				},
-			},
+			Satisfies:     &regularSatisfiesList,
```
Comment (afeld) [inline models/components_test.go]: Much better!

**Comment by afeld** [comment]
Looks good! What does this depend on in #143?

## TODOs

* [ ] Create corresponding pull request on https://github.com/opencontrol/schemas

**Comment by afeld** [comment]
I imagine we'll want to re-examine this PR with all of the recent changes around the schema, yeah?

**Comment by afeld** [comment]
Looks like this is moved to https://github.com/opencontrol/compliance-masonry/pull/160.

**Pull request closed without merging**